aea/channel/gym.py

```diff
@@ -60,7 +60,7 @@ def connect(self, public_key: str) -> Optional[Queue]:
             return None
 
         assert len(self._queues.keys()) <= 1, "Only one public key can register to a gym."
-        q = Queue()
+        q = Queue()  # type: Optional[Queue]
```

Comment:
Why `Optional[Queue]` and not simply `Queue`?

Reply:
Oops. My bad, will fix.